docs/main/faqs/faqs.md

```diff
@@ -167,61 +561,61 @@ No. This may be a feature of future releases.
 
 [Go to top](#top)
 
-### Can replicated engine do async replication to a node in the same cluster? How about a different cluster?
+### Can Replicated Storage do async replication to a node in the same cluster? How about a different cluster?
 
-Replicated engine does not peform asynchronous replication.
+Replicated Storage does not peform asynchronous replication.
 
 [Go to top](#top)
 
-### Does replicated engine support RAID?
+### Does Replicated Storage support RAID?
 
 Replicated Storage pools do not implement any form of RAID, erasure coding or striping. If higher levels of data redundancy are required, replicated volumes can be provisioned with a replication factor of greater than one, which will result in synchronously mirrored copies of their data being stored in multiple Disk Pools across multiple Storage Nodes. If the block device on which a Disk Pool is created is actually a logical unit backed by its own RAID implementation \(e.g. a Fibre Channel attached LUN from an external SAN\) it can still be used within a replicated disk pool whilst providing protection against physical disk device failures.
 
 [Go to top](#top)
 
-### Does replicated engine perform compression and/or deduplication?
+### Does Replicated Storage perform compression and/or deduplication?
 
 No.
 
 [Go to top](#top)
 
-### Does replicated engine support snapshots? Clones?
+### Does Replicated Storage support snapshots? Clones?
 
-No but these may be features of future releases.
+No, but these may be features of future releases.
 
 [Go to top](#top)
 
 ### Which CPU architectures are supported? What are the minimum hardware requirements?
 
-The replicated engine nightly builds and releases are compiled and tested on x86-64, under Ubuntu 20.04 LTS with a 5.13 kernel. Some effort has been made to allow compilation on ARM platforms but this is currently considered experimental and is not subject to integration or end-to-end testing by replicated engine's maintainers.
+The Replicated Storage nightly builds and releases are compiled and tested on x86-64, under Ubuntu 20.04 LTS with a 5.13 kernel. Some effort has been made to allow compilation on ARM platforms but this is currently considered experimental and is not subject to integration or end-to-end testing by Replicated Storage's maintainers.
 
-Minimum hardware requirements are discussed in the [quickstart section](../quickstart-guide/) of this documentation.
+Minimum hardware requirements are discussed in the [quickstart section](../quickstart-guide/installation.md) of this documentation.
 
-Replicated engine does not run on Raspbery Pi as the version of the SPDK. Replicated engine requires ARMv8 Crypto extensions which are not currently available for Pi.
+Replicated Storage does not run on Raspbery Pi as the version of the SPDK. Replicated Storage requires ARMv8 Crypto extensions which are not currently available for Pi.
```

Comment:
as the current version of SPDK requires ARMv8 Crypto extensions...

@tiagolobocastro can you confirm if this limitation needs to be updated?